src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -930,8 +935,10 @@ void java_bytecode_parsert::rbytecode(
       address+=2;
       break;
 
-    default:;
+    default:
+      assert(false && "unknown instruction");
```

Comment:
This should rather output an error and throw an exception if this is the first point where we detect an unknown byte code.